tutorial/numpy-functions.md

```diff
@@ -356,30 +356,7 @@ In Pyccel we try to support the Numpy functions which developers use the most..
     res = matmul(arr,arr)
     print *, 'res: ' // ' ' , res
 
-    end program prog_boo
+    end program prog_test_matmul
     ```
 
-## [Numpy types](https://numpy.org/devdocs/user/basics.types.html)
-
--   Supported types: bool, int, int8, int16, int32, int64, float, float32, float64, complex64 and complex128. They can be used as cast functions too.
-
-    Note: np.bool, np.int and np.float are just aliases to the Python native types, and are considered as a deprecated way to work with Python built-in types in NumPy.
-
-## Other functions
-
--   Supported [math functions](https://numpy.org/doc/stable/reference/routines.math.html) (optional parameters are not supported):
-
-    sqrt, abs, sin, cos, exp, log, tan, arcsin, arccos, arctan, arctan2, sinh, cosh, tanh, arcsinh, arccosh and
-    arctanh.
-
--   Supported [array creation routines](https://numpy.org/doc/stable/reference/routines.array-creation.html) (fully supported):
-
-    -   empty, full, ones, zeros, arange (`like` parameter is not supported).
-    -   empty_like, full_like, and zeros_like, ones_like (`subok` parameter is not supported).
-    -   rand, randint.
-
--   others:
-
-    amax, amin, sum, shape, floor
-
```

Comment:
Ehm... I did not ask to move this whole section to file `ndarrays.md`...

What I said is that methods and properties of the `numpy.NDArray` class **should be listed in** `ndarrays.md`. Further, these methods/properties should point to the corresponding functions in file `numpy-functions.md`.

Anyway, I agree that the list of supported data types belong to `ndarrays.md`. Can you check if there was such a list already? Maybe it just needs updating.

Reply:
Fixed in 74f97fd (but the diff is unhelpful)